frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskLogs.tsx

```diff
@@ -37,6 +40,7 @@ class PipelineTaskLogs extends React.Component<PipelineTaskLogsProps, PipelineTa
 
   constructor(props) {
     super(props);
+    this.downloadLogs.bind(this);
```

Comment:
Unnecessary when using this notation `downloadLogs = () => {`

Reply:
moved that out